### PR TITLE
Ets 2199 prevent html deletion

### PR DESCRIPTION
**Pull request by BirgitPohl**
## 0.45.2
- Prevents HTML element deletion on ellipsis

**Todo**: Slight merge conflicts are incoming.

Evaluate, if HTML-Ellipsis is doing a good job:
Screenshot 2019-07-26 at 15 43 27
If not, we may want to borrow its code and extend some functionality.

### CODE REVIEW

**Review by xavero-evandro — DISMISSED** [review]

**Review by susannegernandt — DISMISSED** [review]

**Comment by dknx01** [comment]
Do I see it right, that in the teaser.js (nearly) all functions are global? What happen if we have for example a "getFontSize" function in another file, too?

File: package.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
     "access": "public"
   },
   "dependencies": {
-    "core-js-bundle": "^3.0.0-beta.13"
+    "core-js-bundle": "^3.0.0-beta.13",
```
Comment (dknx01) [inline package.json:33]: Where do we need the core-js? In the changes here i don't see it at the first look?

Comment (BirgitPohl) [reply package.json:33]: Inside the main.js. And  they have good stable versions. I think, we could update it.

Comment (BirgitPohl) [reply package.json:33]: Btw, It is marked as a change, because a comma was added, because a new element has been added to this json-structure. :)

**Pull request merged**